src/component/AddItem.js

```diff
@@ -19,6 +20,8 @@ const AddItem = ({ firestore }) => {
 
   //   Write item to Firebase setting uniqueToken as document name
   const addItem = (normalizedName, numberOfDays) => {
+    let nextPurchase = new Date();
+    nextPurchase.setDate(today.getDate() + parseInt(numberOfDays));
```

Comment:
Wait...I saw this in the other PR, too. Was this branch based on that one? That's important info to note in the PR description when it's the case. Thanks!

Reply:
Ethan and I made a not so good decision earlier this week. Because we got started on our branch a bit late, we decided to copy and paste the changes that the other team had made to their branch thus far into our branch. This was in hopes of using what they had already created in regards to adding new info to the DB and trying to minimize merge conflicts in the future. After messing around with the code, we realized we didn't need much of what they had done after all... live and learn. Sorry for the confusion!

Reply:
All good! Live and learn 🙂